### PR TITLE
45503 implement AreaButler in detail page template

### PR DESCRIPTION
**Pull request by dai-eastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/748
changed log:
Add AreaButler on detail page template

### CODE REVIEW

File: css/onoffice-default.css [hunk 1]
```diff
@@ -71,4 +71,9 @@ form[data-applicant-form-id] #spinner {
 
 .message{
     display: none!important;
+}
+
+.oo-area-butler {
+    position: relative;
+    height: 500px;
 }
```
Comment (andernath) [inline css/onoffice-default.css:79]: I think this fits more in onoffice-defaultview.css
Whats your opinion?

**Review by andernath — COMMENTED** [review]
@dai-eastgate Please take a look at my suggestion

**Comment by dai-eastgate** [comment]
@andernath I will check it

**Comment by dai-eastgate** [comment]
@andernath 
- I found that the issue https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/235 mentioned not adding CSS to the onoffice_defaultview.css file because it can lead to the breaking of personalized templates.
=> So please check the information before you want to move the CSS to the onoffice_defaultview.css file. If you still want to update the CSS in this file, I will fix it.
- Please explain the disadvantages of placing CSS in onoffice-default.css

**Comment by andernath** [comment]
> * I found that the issue [P#71173 Make CSS local to templates #235](https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/235) mentioned not adding CSS to the onoffice_defaultview.css file because it can lead to the breaking of personalized templates.
>   => So please check the information before you want to move the CSS to the onoffice_defaultview.css file. If you still want to update the CSS in this file, I will fix it.

Oh Thanks for investigating! I see we should not use onoffice_defaultview.css neither onoffice-default.css
Instead of template.dist/onoffice-style.css seems to be the right one. Between /* details */ and /* responsive */ codeblock.

> * Please explain the disadvantages of placing CSS in onoffice-default.css

I got suspicious because of the file naming and classes inside. But anyway, thanks to your hint, we've found out where we need to insert styling changes: 
template.dist/onoffice-style.css


Don't worry. **I will add this change.** So Frederic can start testing. :)

**Review by andernath — APPROVED** [review]

**Comment by dai-eastgate** [comment]
@andernath For this new AreaButler feature, placing the CSS in template.dist/onoffice-style.css makes sense, but for some old features, be careful placing the CSS in template.dist/onoffice-style.css because this may affect backward compatibility

**Comment by fredericalpers** [comment]
@dai-eastgate After reviewing there are a few changes that have to be made.

- [ ] Add a H2-Headline with the text `AreaButler`
- [ ] Add styling as for the other sections `padding-bottom: 15px; padding-top: 15px; border-bottom: 2px solid #efefef;`

**Comment by dai-eastgate** [comment]
> * [x]  Add a H2-Headline with the text `AreaButler`
> * [x]  Add styling as for the other sections `padding-bottom: 15px; padding-top: 15px; border-bottom: 2px solid #efefef;`

@fredericalpers I have updated CSS. This is video demo:

https://github.com/onOffice-Web-Org/oo-wp-plugin/assets/106214469/8f8fec0e-1245-403c-9e6f-c3051b40cbe5

Note: Currently I cannot access https://areabutler.dev.areabutler.de/ on the onOffice Enterprise Enterprise so the map for AreaButler is not displayed, instead it is showing as "loading"
Please check and let me know your opinion. Thanks!

**Review by andernath — APPROVED** [review]

**Pull request merged**